frontend/src/locales/en-CA.json

```diff
@@ -48,6 +48,39 @@
         "unableToLoad": "Unable to load permit or project, please try again later",
         "updateHistory": "Update history"
       },
+      "projectTeamAddModal": {
```

Comment:
Can't remember where we landed with handling common strings in i18 but there are a few common ones between your components